.travis.yml

```diff
@@ -1,6 +1,16 @@
 language: elixir
 elixir:
   - 1.3.2
-  - 1.4.2
+  - 1.4.4
+  - 1.5.1
 otp_release:
   - 19.0.2
+  - 20.0
+
+matrix:
+  exclude:
+    - elixir: 1.3.2
+      otp_release: 20.0
+    - elixir: 1.4.4
+      otp_release: 20.0
```

Comment:
the hex.pm repo doesn't seem to have any 1.4/otp-20 builds available (403 responses on every version in travis), but that combo should be supported just fine